Libs/Widgets/ctkAxesWidget.cpp

```diff
@@ -258,7 +258,7 @@ void ctkAxesWidget::paintEvent(QPaintEvent *)
     {
     //if (d->HighlightAxes)
       {
-      QPen pen;
+      QPen pen(this->palette().color(QPalette::Active, QPalette::Highlight));
```

Comment:
I'm not sure to understand why you highlight axis even if it is not a highlighted axis...
Are you sure about Highlight ? Isn't it WindowText instead ?